pkg/smartcontract/parameter.go

```diff
@@ -387,6 +387,10 @@ func NewParametersFromValues(values ...interface{}) ([]Parameter, error) {
 func ExpandParameterToEmitable(param Parameter) (interface{}, error) {
 	var err error
 	switch t := param.Type; t {
+	case Hash160Type:
+		return param.Value.(util.Uint160).BytesBE(), nil
```

Comment:
Why don't you handle them in `stackitem.Make`? `emit.Array` can work with Uint* types already, it doesn't need this code and it can be usefule for other `stackitem.Make` uses to be able to take Uint* directly.

Reply:
Fixed.